src/main/java/com/internet/shop/dao/impl/OrderDaoImpl.java

```diff
@@ -0,0 +1,65 @@
+package com.internet.shop.dao.impl;
+
+import com.internet.shop.dao.OrderDao;
+import com.internet.shop.db.Storage;
+import com.internet.shop.library.Dao;
+import com.internet.shop.models.Order;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+@Dao
+public class OrderDaoImpl implements OrderDao {
+
+    @Override
+    public Order create(Order order) {
+        Storage.addOrder(order);
+        return order;
+    }
+
+    @Override
+    public Optional<Order> getById(Long id) {
+        return Storage.orders.stream()
+                .filter(order -> order.getId()
+                        .equals(id))
```

Comment:
```suggestion
                .filter(order -> order.getId().equals(id))
```
не варто розтягувати те, що можна помістити в один рядок)
виклик кожного нового метода стріма - доцільно переносити на новий рядок, а виклик внутрішніх методів - ні (хіба що у випадку коли в нас є складна структура виклику внутрішніх методів)
відповідно це ж саме стосується решти методів